### ISSUE TITLE
Error: access denied. Please try again later 

### ISSUE THREAD

**Issue by Todd1561** [thedmd/pianobar-windows#36]
### Subject of the issue
Been getting the error message in the title on and off for the last 2-3 weeks. Maybe 60% of the time. I’m able to log in fine but when I pick a playlist I’ll get the error.

Version 2020.04.20
Windows server 2012r2

Stock config file other than credentials 

### Steps to reproduce
Just pick a playlist 

any thoughts?

**Comment by stevenwnt** [comment]
i had the same problem it seems player = mf aka "Windows Media Foundation" backend
try player = ds aka "Direct Show" backend
all you have to do it open your pianobar.cfg and put player = ds as a new line

**Comment by Todd1561** [comment]
Thanks for that, I might try that out soon.  I guess I forgot to update this thread.  I dug into it more and it seems to be an upstream Pandora issue, nothing to do with Pianobar.  I narrowed it down to just a couple of my lesser used stations and when I tried to play them on the official Pandora iOS app it would fail with the same message.  At that point I just shrugged it off as a Pandora problem and deleted the couple affected stations I had.
